hack/update-codegen.sh

```diff
@@ -491,6 +491,7 @@ function codegen::openapi() {
         k8s_tag_files_matching \
             vendor/k8s.io/apiextensions \
             vendor/k8s.io/api/autoscaling/v1 \
+            vendor/k8s.io/apimachinery/pkg/api/resource \
```

Comment:
resource.Quantity is referenced by the `autoscaling` generated openapi. Required its definitions for the generated openapi to be complete.